<!-- Launch file for running the caught robber node on a computer.
See wiki for instructions on running caught robber node on Odroid - LT-->

<launch>
  <arg name="catch_dist" default="0.7" />
  <arg name="cops" default=["pris"] />
  <arg name="num_cops" default="1" />
  <arg name="num_robbers" default="1" />
  <arg name="show_mask" default="False" />
  <arg name="show_cop_feed" default="False" />
  <arg name="image_proc" default="False" />

  <node name="caught_robber" pkg="caught" type="caught_robber_comp.py" output="screen">
    <param name="dist" value="$(arg catch_dist)" />
    <param name="show_mask" value="$(arg show_mask)" />
  </node>

  <!-- to pass a list of cops see the following line
  <rosparam param="cops">["pris"]</rosparam>
-->

  <!-- Test caught robber node? Uncomment following
  <node name="test_caught" pkg="caught" type="test_caught.py" output="screen">
    <param name="num_robbers" value="$(arg num_robbers)" />
  </node>
-->

  <!-- if cops's 3dsensor.launch "rgb_processing" argument is false
  ie ... <arg name="rgb_processing" value="false"/>
  then, copy and uncomment the following line for each cop
  <node name="image_proc" pkg="image_proc" type="image_proc" ns="/<cop_name>/camera/rgb"/>
  -->

  <!-- To view cop's image feed,
  copy, uncomment and CHANGE NAME of following node for each cop:
  <node name="image_view" pkg="image_view" type="image_view">
    <remap from="image" to="/<cop_name>/camera/rgb/image_color"/>
  </node>
-->

</launch>
